packages/directory/addon/templates/components/dir-sidebar.hbs

```diff
@@ -0,0 +1,26 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<section class='dir-sidebar__groups'>
+  {{#each directories as | directory |}}
+    <div
+      class='dir-sidebar__group {{if (eq selectedDirectory directory) 'dir-sidebar__group--selected'}}'
+      onclick={{pipe (action (mut selectedDirectory) directory) (action (mut selectedFilter) (object-at 0 directory.filters))}}
```

Comment:
is mut selectedFilter required?

Reply:
Yes, each directory has its set of filters. Its not necessary given that the selected filter would exist for the selected directory. So resetting it.